python/cudf/cudf/_fuzz_testing/tests/fuzz_test_csv.py

```diff
@@ -40,7 +40,7 @@ def csv_writer_test(pdf):
 @pythonfuzz(
     data_handle=CSVWriter,
     params={
-        "sep": list([",", "|", "\t", "\t\t", "~"]),
+        "sep": list([",", "|", "\t", "\r", "~"]),
```

Comment:
why this change?

Reply:
Had to make this change because we don't support multi-characters in `sep`: https://github.com/rapidsai/cudf/pull/7095